content/en/contribute/code/core/fixing-e2e-tests.md

```diff
@@ -49,10 +49,6 @@ Then grep for just the tests you want to run:
 protractor /home/kenn/webapp/tests/e2e.tests.conf.js --specs='/home/kenn/webapp/tests/e2e/api/controllers/_changes.spec.js' --grep="should allow DB admins to POST to _changes"
 ```
```

Comment:
I am not sure how to handle this `Option 2` section. 
@ngaruko can you please advise?